tutorials/parquet-catalog-demos/euclid-hats-parquet.md

```diff
@@ -0,0 +1,235 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Euclid Quick Release 1: MER Catalogs in HATS Parquet
+
++++
+
+## Learning Goals
+By the end of this tutorial, you will:
+
+- # [TODO]
+
++++
+
+## Introduction
+
++++
+
+This notebook explores the HATS version of the [Euclid Q1](https://irsa.ipac.caltech.edu/data/Euclid/docs/overview_q1.html) MER Catalogs that has been created by IRSA.
+In this version, the three MER Catalogs (MER, MER Morphology, and MER Cutouts) have been joined by Object ID into a single Parquet dataset.
+It contains 29,953,430 rows and 601 columns. The total size is 32 GB.
+The dataset is partitioned following [HATS](https://hats.readthedocs.io/) (Hierarchical Adaptive Tiling Scheme).
+
++++
+
+## Installs and imports
+
+```{code-cell}
+# !pip install 'hats>=0.5' 'lsdb>=0.5' matplotlib numpy s3fs
+```
+
+```{code-cell}
+import os
+
+import dask.distributed
+import hats
+import lsdb
+import matplotlib.colors
+import matplotlib.pyplot as plt
+import numpy as np
+```
+
+## 1. Setup
+
+```{code-cell}
+# Need UPath for the testing bucket. Otherwise hats will ignore the credentials that Fornax
```

Comment:
Until the data is fully public we should definitely ignore this notebook from any CI execution. I'll push changes for that directly to your branch and also add the labels accordingly, so parts of CI is not even triggered.